### ISSUE TITLE
Sep 28 - Oct 2: Build Ethereum Smart Contracts Using Solidity

### ISSUE THREAD

**Issue by meaghanlewis** [microsoft/ReactorPlanning#61]
# Event Information
| Category | Details |
|-----------|---------|
| Reactor Topic | Emerging Tech |
| Monthly Topic (if applicable) | Skill Up September |
| Weekly Topic (if applicable) | Blockchain |
| Stream Date | Sep 28 - Oct 2, 2020 | 
| Stream Duration | 90 min | 
| Title | Build Ethereum Smart Contracts Using Solidity |
| Description | <p>Smart contracts are code that verify and enforce behavior on the Ethereum platform. They regulate the transfer of digital assets and currency. In this workshop, you'll use the programming language Solidity (https://solidity.readthedocs.io/) to develop your own smart contracts.</p><p>Prerequisites: You should understand blockchain principles and want to learn best practices for developing your own smart contracts.</p><p>To follow along during the workshop, install Visual Studio Code (https://code.visualstudio.com/Download).</p> |
| Content | https://github.com/microsoft/Reactors/tree/master/Blockchain_1/Part_5_Building_Smart_Contracts_For_Blockopoly |

Additional Resources:  
<Additional resources that can be linked from social media for promotion or added to the description of VOD>

# Workshop details
| Date |  Local Start Time |  Location |  Speaker | SharePoint ID |
|----- |------------------|-----------|----------|--------------|
| Sep 28 |  4:00 PM GMT+4 | Abu Dhabi | Monette Stephens  | 7941 |
| Sep 28 |  6:00 PM  GMT+3 | Tel Aviv | NO STREAM | N/A |
| Sep 29 |  10:00 AM  GMT+10| Sydney | Monette Stephens | 7943 |
| Sep 29 |  6:30 PM GMT+5:30| Bangalore | Monette Stephens | 7944 |
| Sep 30 |  3:30 PM GMT-3| Sao Paulo | Solange Guieros | 7945 |
| Sep 30 |  7:30 PM GMT+8 | Shanghai | | 7946 |
| Sep 30 | 6:00 PM GMT+1  | London/Stockholm | Monette Stephens | 7947 |
| Sep 30 | 4:00 PM PDT | US/Canada | Monette Stephens | 7948 |

# Event Checklist

## 10 Weeks Prior
- [X] Reactor Content Team (RCT) opens a new issue for an upcoming local workshop filling in the Event Information and Workshop details tables. RCT member who owns content (content owner) this week will be the assignee.
- [x] RCT and content owner discuss details of the event in the comments of this issue. RCT adds a _confirmed-event_ label and _local-digital-event_ tag to the issue and adds event to the __Local Digital Events__ project. Issue will appear in the __Upcoming Streams__ column
- [x] RCT adds a _ready-for-marketing-edit_ label so the RCT can review title and description

## 8 Weeks Prior
- [x] RCT does a review of title and description and removes the _ready-for-marketing-edit_ label 
- [x] RCT moves the issue to the __Ready for SharePoint__ column
- [ ] RCT informs the Reactor Marketing Team of the new event information
- [ ] RCT confirms the speaker
- [ ] RCT provides speaker with assets (e.g. slides, overlays) and the survey data for the event
- [ ] RCT moves the issue to the __Content Preparation In Progress__ column

## 5 weeks Prior
- [ ] Content owner opens a PR with the new content for the workshop on the [customer-facing Reactor repository](https://github.com/microsoft/Reactors) and adds the _content-development_ and _local-digital-event_ tag to the PR. 
- [ ] Content owner adds to the same PR modification to the [Streaming README](https://github.com/microsoft/Reactors/tree/main/Online) 
- [ ] RCT reviews, approves, and merges the changes
- [ ] RCT shares content with the vendor/instructor
- [ ] RCT provides vendor/instructor with assets (e.g. slides, overlays) and the survey data for the event
- [ ] RCT ensures that instructors have received an invite to the workshops and that PM team works with instructors to get setup in the platform (e.g. Teams, Teams Live Event)

## 1 Week Prior
- [ ] Connect vendor/instructor with the PM(s) for the locations where this event has been marketed
- [ ] Check-in with vendor/instructor that they are prepared for the workshop
- [ ] RCT moves the issue to the __Ready to Stream__ column

## During and After Stream
- [ ] Instructor (and optional moderator) run the stream
- [ ] _(Optional)_RCT submits recording to be uploaded to the Reactor YouTube and to be made available on LearnTV
- [ ] RCT moves issue to the __Stream Complete__ column

**Comment by mebaumb** [comment]
@meaghanlewis - do we have a publishable title & abstract for these workshops?

**Comment by meaghanlewis** [comment]
👋  @KimberlyBolton1 I just updated the description for this workshop. Could you please take a pass at it and edit?

**Comment by KimberlyBolton1** [comment]
@meaghanlewis edited version below. let me know if you have any changes:

Build Smart Contracts Using Solidity and Ethereum 
Smart contracts are computer protocols that verify and enforce the performance of a given contract. In this workshop, you’ll use the programming language Solidity (https://solidity.readthedocs.io/) to code smart contracts for the Ethereum computing platform. 
Prerequisites: You should understand blockchain principles and want to learn best practices for developing your own smart contracts.
To follow along during the workshop, install Visual Studio Code (https://code.visualstudio.com/Download).

**Comment by meaghanlewis** [comment]
Thanks @KimberlyBolton1! I'd like to make the definition of smart contracts a bit simpler.

How about:
**Build Ethereum smart contracts using Solidity**
Smart contracts are code that verify and enforce behavior on Ethereum. They regulate the transfer of digital assets and currency. In this workshop, you'll use the programming language Solidity (https://solidity.readthedocs.io/) to develop your own smart contracts.
Prerequisites: You should understand blockchain principles and want to learn best practices for developing your own smart contracts.
To follow along during the workshop, install Visual Studio Code (https://code.visualstudio.com/Download).

**Comment by KimberlyBolton1** [comment]
@meaghanlewis Slight edit to the first sentence, but otherwise looks good. 
@mebaumb abstract is final and moved to Ready for SharePoint.

**Comment by mebaumb** [comment]
Abu Dhabi, Sydney, Bangalore, Sao Paulo, Shanghai, London, & US/Canada have been published

**Comment by mebaumb** [comment]
These events have been delivered, so I am closing the issue!
